pages/auth/login.vue

```diff
@@ -68,8 +68,22 @@
             {{ $t('auth.login.label') }}
           </hc-button>
         </form>
-        <p class="subtitle is-6">{{ $t('quotes.african.txt') }}</p>
-        <p class="subtitle is-7">{{ $t('quotes.african.author') }}</p>
+        <p class="subtitle is-6">
+          {{ $t('quotes.african.txt') }}
+          <div style="padding-top: 10px">
```

Comment:
This is dirty... Can somebody point me to the right css class or where to create a new one for this purpose?
@roschaefer

Reply:
Sure, just take a look at the other components. You can add a class name on the element and then style it inside the style tag on the component or page.

Reply:
![image](https://user-images.githubusercontent.com/21975779/45600344-8854f880-b9fb-11e8-98e9-325bf9dd33ac.png)
That is the current status.

Reply:
> Sure, just take a look at the other components. You can add a class name on the element and then style it inside the style tag on the component or page.

Yeah, that's what I tried. But when I add .subtitle.is-8 as a copy of .subtitle.is-7 plus the additional `padding-top: 10px;` and change the class in the login.vue to `subtitle is-8`, it is not recognized and will be displayed with default style...

Reply:
Okay, I see that you are using `div` elements inside of `p`, that’s not valid. Try to use `span` instead. 

And the Stylesheet is inside of the component you are workin on. Ignore everything inside the node modules folder. That are third party styles that not all are applied. 

When you are searching for global styles, the are not .css but .scss files. The local styles live inside the .vue files then selve in an Style Tag. See https://vuejs.org/v2/guide/single-file-components.html

Reply:
I would stick with local styling in this case or see the [https://bulma.io/](Bulma) documentation for the available styles.